src/test-utilities/tests/legacy.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {mountWithAppProvider} from '../legacy';
+
+describe('mountWithAppProvider', () => {
+  let consoleSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    consoleSpy = jest.spyOn(console, 'error');
+    consoleSpy.mockImplementation(() => {});
```

Comment:
Silences the error

Reply:
Which error is that?

Reply:
The one at the bottom of this file around `expect(consoleSpy).toHaveBeenCalledWith(`. This mockImplementation makes it so the error we're deliberately causing does not get output to the console as we expect it to appear.